src/app/features/event/event-card/event-card.component.scss

```diff
@@ -0,0 +1,46 @@
+@use '@angular/material' as mat;
+@use '../../../styles/palettes' as palette;
+
+.event-card {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    padding: 4px;
+    max-width: 560px;
+    background-color: mat.get-color-from-palette(palette.$gray-palette, 500);
+    border: 1px solid mat.get-color-from-palette(palette.$blue-palette, 500);
+    border-radius: 10px;
+    box-shadow: 1px 4px 4px mat.get-color-from-palette(palette.$gray-palette, 900);
+    &__info {
+        display: flex;
+        gap: 16px;
+    }
```

Comment:
propozycja: dodałbym tutaj `padding: 8px;` żeby miało więcej miejsca (wiem że na figmie to wygląda inaczej, ale moim zdaniem wygląda nieco lepiej z tym paddingiem, ponieważ nie jest aż tak przyklejone do krawędzi)

Reply:
Done.